test/metabase/driver/clickhouse_test.clj

```diff
@@ -175,3 +176,17 @@
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime")))
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime64")))
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime64(3)")))))
+
+(deftest clickhouse-insert
+  (mt/test-driver
+   :clickhouse
+   (t2.with-temp/with-temp
+     [:model/Database db
+      {:engine  :clickhouse
+       :details (tx/dbdef->connection-details :clickhouse :db {:database-name "default"})}]
+     (driver/drop-table!   :clickhouse (:id db) "insert_test")
+     (driver/create-table! :clickhouse (:id db) "insert_test" {:id "Int64", :name "String"})
```

Comment:
I recommend using a finally block to ensure that the table is dropped at the end.

Reply:
Sure fixed 8eff462b75fa74784fec98eef1239a063052ab2e